README.md

```diff
@@ -1 +1,38 @@
-# Group_E_Simulation_Life_Cycle
\ No newline at end of file
+# Group_E_Simulation_Life_Cycle
+Françoise Blanc
+Roman Cardenas Rodriguez
+Ghida Dandan
+Aidan Fahlman
+Bruno St-Aubin
+
+## Description
+Need to Develop this Section
```

Comment:
Is it OK if we use the same text as the Deliverable 1?

Reply:
That is what I was considering doing but left it blank for the time being. Wanted to make sure everyone was ok with this

Reply:
My bet is it will be ok. We'll probably hear more about this in the upcoming sync lectures.

Reply:
Bruno already addressed this. I'm resolving the conversation.